infrastructure/code/code_test.go

```diff
@@ -75,11 +75,16 @@ func TestCreateBundle(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
-			_, missingFiles, err := c.createBundle(context.Background(), "testRequestId", dir, []string{file}, []string{})
+			bundle, err := c.createBundle(context.Background(),
+				"testRequestId",
+				dir,
+				[]string{file},
+				map[string]bool{})
 			if err != nil {
 				t.Fatal(err)
 			}
-			assert.Len(t, missingFiles, 1, "bundle should have 1 missing files")
+			bundleFiles := bundle.Files
+			assert.Len(t, bundleFiles, 1, "bundle should have 1 bundle files")
```

Comment:
missing files != bundle files

Reply:
It's supposed to be "bundle files". The missing files are coming from a component that is not under test here (the snyk code client is mocked). It always tested for bundle files, nothing was changed logic-wise